#---
#
# Solves build issue when compiling a LLVM or GCC compiler:
#
# sanitizer_platform_limits_posix.cc:874:46: error: ‘SCSI_IOCTL_TAGGED_DISABLE’ was not declared in this scope
# unsigned IOCTL_SCSI_IOCTL_TAGGED_DISABLE = SCSI_IOCTL_TAGGED_DISABLE;
# sanitizer_platform_limits_posix.cc:875:45: error: ‘SCSI_IOCTL_TAGGED_ENABLE’ was not declared in this scope
# unsigned IOCTL_SCSI_IOCTL_TAGGED_ENABLE = SCSI_IOCTL_TAGGED_ENABLE;
#
#----
diff -uprN ../../work.orig/aarch64-unknown-linux-gnueabi/aarch64-unknown-linux-gnueabi/sysroot/usr/include/scsi/scsi.h ./aarch64-unknown-linux-gnueabi/sysroot/usr/include/scsi/scsi.h
--- ../../work.orig/aarch64-unknown-linux-gnueabi/aarch64-unknown-linux-gnueabi/sysroot/usr/include/scsi/scsi.h	2017-08-03 01:08:13.000000000 +0000
+++ ./aarch64-unknown-linux-gnueabi/sysroot/usr/include/scsi/scsi.h	2023-03-06 23:41:00.328424414 +0000
@@ -264,6 +264,8 @@ struct ccs_modesel_head {
 #define SCSI_IOCTL_GET_IDLUN		0x5382
 
 /* 0x5383 and 0x5384 were used for SCSI_IOCTL_TAGGED_{ENABLE,DISABLE} */
+#define SCSI_IOCTL_TAGGED_ENABLE		0x5383
+#define SCSI_IOCTL_TAGGED_DISABLE		0x5384
 
 /* Used to obtain the host number of a device. */
 #define SCSI_IOCTL_PROBE_HOST		0x5385
